diff --git a/node_modules/semver/semver.js b/node_modules/semver/semver.js
index 636fa43..44975af 100644
--- a/node_modules/semver/semver.js
+++ b/node_modules/semver/semver.js
@@ -373,7 +373,7 @@ function SemVer (version, options) {
 SemVer.prototype.format = function () {
   this.version = this.major + '.' + this.minor + '.' + this.patch
   if (this.prerelease.length) {
-    this.version += '-' + this.prerelease.join('.')
+    this.version += '-' + this.prerelease.join()
   }
   return this.version
 }
